yt/yt/server/master/security_server/access_log.cpp

```diff
@@ -278,6 +280,23 @@ bool IsAccessLoggedType(const EObjectType type)
     return typesForAccessLog.contains(type);
 }
 
+bool IsAccessLogEnabled(NCellMaster::TBootstrap* bootstrap) {
```

Comment:
You can check with the code-style located here: 
https://github.com/ytsaurus/ytsaurus/blob/main/yt/styleguide/cpp.md

In particular, the opening bracket should be on the following line.